pyrometheus/codegen/python.py

```diff
@@ -335,23 +368,24 @@ def get_species_enthalpies_rt(self, temperature):
             % endfor
             ])
 
-    def get_species_entropies_r(self, temperature):
+    def get_species_entropies_r(self, pressure, temperature):
         return self._pyro_make_array([
             % for sp in sol.species():
-            ${cgm(ce.poly_to_entropy_expr(sp.thermo, "temperature"))},
+            ${cgm(ce.poly_to_entropy_expr(sp.thermo, "temperature"))}
+            - self.usr_np.log(pressure/101325.0),
             % endfor
             ])
 
-    def get_species_gibbs_rt(self, temperature):
+    def get_species_gibbs_rt(self, pressure, temperature):
         h0_rt = self.get_species_enthalpies_rt(temperature)
-        s0_r = self.get_species_entropies_r(temperature)
+        s0_r = self.get_species_entropies_r(pressure, temperature)
         return h0_rt - s0_r
 
-    def get_equilibrium_constants(self, temperature):
+    def get_equilibrium_constants(self, pressure, temperature):
         rt = self.gas_constant * temperature
-        c0 = self.usr_np.log(self.one_atm / rt)
+        c0 = self.usr_np.log(pressure / rt)
 
-        g0_rt = self.get_species_gibbs_rt(temperature)
+        g0_rt = self.get_species_gibbs_rt(pressure, temperature)
```

Comment:
Here, and in the lines that follow, let's make sure we have consistency in the way thermo variables are ordered as function arguments. (It may be correct already.)

Reply:
:+1:

Reply:
Ok, thanks for clarifying. We should modify `get_density` for consistency then.